src/formpack/pack.py

```diff
@@ -337,7 +337,7 @@ def to_json(self, **kwargs):
     def export(self, lang=UNSPECIFIED_TRANSLATION, group_sep='/', hierarchy_in_labels=False,
                versions=-1, multiple_select="both",
                force_index=False, copy_fields=(), title=None,
-               tag_cols_for_header=None):
+               tag_cols_for_header=None, lang_header=-1):
```

Comment:
```suggestion
               header_lang=None,
               tag_cols_for_header=None):
```

Reply:
I think we cannot set None to default value for "header_lang" otherwise we cannot know if it is not set or set to None. And, if user call with : (lang=False,header_lang=None) I don't want to change value of arg header_lang. If user call with (lang=False) - header_lang is not set, I want to override header_lang with lang value for not change the current algo.

Reply:
> Hi @jdugh . Thanks for the PR. Could you add a test that uses this column and shows how the values would change when this parameter is used?
> 
> Also, if I understand correctly what this does, it might make more sense to call it `header_lang` or `header_language`.

Ok thank you. I go to create the test and change the name to `header_lang`. Here and in KPI views.

Reply:
Ah yes. Thanks for reminding me of this. I will discuss this with the team and see if we can agree on `-1` or a different solution.

Reply:
hi @dorey , do you know if you can agree a `-1` value (or a different solution), and also, accept this PR in the futur ?